packages/mobile/src/paymentRequest/IncomingPaymentRequestListItem.tsx

```diff
@@ -42,7 +42,7 @@ export class IncomingPaymentRequestListItem extends React.Component<Props> {
       firebasePendingRequestUid: id,
     }
 
-    if (addressValidationType !== AddressValidationType.NONE) {
+    if (addressValidationType && addressValidationType !== AddressValidationType.NONE) {
```

Comment:
Looks like this caught a possible bug, @tarikbellamine is this still correct?

Reply:
Looks ok to me :)